src/paradigma/util.py

```
@@ -337,6 +342,11 @@ def aggregate_parameter(parameter: np.ndarray, aggregate: str) -> np.ndarray | i
         return np.mean(parameter)
     elif aggregate == 'median':
         return np.median(parameter)
+    elif aggregate == 'mode' and bin_edges is not None:
```

If aggregate equals `mode` and bin_edges equals `None` you probably want to raise an error that bin_edges should be defined when using mode. Now, this if-statement will pass and end up at `raise ValueError(f"Invalid aggregation method: {aggregate}")`, which is not exactly correct I think.

---

Good point. If bin_edges equals `None`, the function will now continue using the other function for computing the mode (which is used for modal pulse rate), which will not return correct values for tremor power. To make this distinction more clear, I changed the aggregate 'mode' to 'mode_binned' for tremor power. If no bin_edges are provided, it will raise an error that these should be defined.